.github/workflows/spelling-check-bot.yml

```
@@ -29,20 +29,28 @@ jobs:
           echo "$output" >> $GITHUB_ENV
           echo "EOF" >> $GITHUB_ENV
 
-      - name: Create an issue
+      - name: Report issues
         if: env.OUTPUT != ''
-        uses: dacbd/create-issue-action@main
-        with:
-          token: ${{ secrets.GITHUB_TOKEN }}
-          title: Weekly spelling check
-          body: |
-            Typos and unknown words:
-
-            ```
-            ${{ env.OUTPUT }}
-            ```
-
-            > [!TIP]
-            > To exclude words from the spellchecker, you can add valid words (web technology terms or abbreviations) to the [terms-abbreviations.txt](https://github.com/mdn/content/blob/main/.vscode/terms-abbreviations.txt) dictionary for IDE autocompletion. To ignore strings that are not words (`AABBCC` in code, for instance), you can add them to [ignore-list.txt](https://github.com/mdn/content/blob/main/.vscode/ignore-list.txt).
+        run: |
+          log_data=$(cat<<EOM
+          Typos and unknown words:
+
+          ~~~
+          ${OUTPUT}
+          ~~~
+
+          > [!TIP]
+          > To exclude words from the spellchecker, you can add valid words (web technology terms or abbreviations) to the [terms-abbreviations.txt](https://github.com/mdn/content/blob/main/.vscode/terms-abbreviations.txt) dictionary for IDE autocompletion. To ignore strings that are not words (\`AABBCC\` in code, for instance), you can add them to [ignore-list.txt](https://github.com/mdn/content/blob/main/.vscode/ignore-list.txt).
+          EOM
+          )
+
+          gh api \
+            --method PATCH \
+            -H "Accept: application/vnd.github+json" \
+            -H "X-GitHub-Api-Version: 2022-11-28" \
+            /repos/mdn/content/issues/35634 \
```

- Bot created a new #35685 issue today.
```suggestion
            /repos/mdn/content/issues/35685 \
```